storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -4214,6 +4203,31 @@ private void maybeAddPlaceholderBoltAggStats(TopologyPageInfo topoPageInfo, Stor
         }
     }
 
+    private CommonAggregateStats getPlaceholderCommonAggregateStats(Object component) {
+        // common aggregate
+        CommonAggregateStats commonStats = new CommonAggregateStats();
+
+        // get num_executors
+        int numExecutors = 0;
+        try {
+            numExecutors = StormCommon.numStartExecutors(component);
+        } catch (InvalidTopologyException e) {
```

Comment:
It looks like we can remove `throws InvalidTopologyException` from `numStartExecutors` and then we don't need to catch the `InvalidTopologyException`

Reply:
Leaving as is